common-fragment-V1/src/main/java/org/pentaho/hadoop/shim/common/DistributedCacheUtilImpl.java

```diff
@@ -184,8 +181,11 @@ public void installKettleEnvironment( FileObject pmrArchive, FileSystem fs, Path
     out.close();
 
     stageForCache( extracted, fs, destination, true, false );
-    java.nio.file.Path kettleHomeDir = Paths.get( System.getProperty( "karaf.home" ) ).getParent().getParent();
-    stagePentahoHadoopShims( fs, destination, kettleHomeDir );
+
+    java.nio.file.Path shimDriverInstallationDirectory =
+      Paths.get( System.getProperty( Const.SHIM_DRIVER_DEPLOYMENT_LOCATION, DEFAULT_DRIVERS_DIR ) );
```

Comment:
Are we going to run into the same problem with windows paths that we fixed other places using this Paths library here?

Reply:
Nope. Tested it out on windows.